tests/components/husqvarna_automower/__init__.py

```diff
@@ -7,6 +7,6 @@
 
 async def setup_integration(hass: HomeAssistant, config_entry: MockConfigEntry) -> None:
     """Fixture for setting up the component."""
+    await hass.config.async_set_time_zone("UTC")
```

Comment:
I'd suggest to choose another default time zone than UTC, also add a comment explaining why we lock down the timezone in the tests.

Reply:
Done in https://github.com/home-assistant/core/pull/121983/commits/d3f00f91b650509d6b0cb24d0de47407c959e8cd